C7GameData/TerrainType.cs

```diff
@@ -17,6 +17,7 @@ public class TerrainType
         public int baseShieldProduction {get; set; }
         public int baseCommerceProduction {get; set; }
         public int movementCost {get; set; }
+        public bool allowCities { get; set; }
```

Comment:
Minor, but it might be nice to manually set the getter so that instead of `IsAllowCities` it reads something like `AllowsCities`.  This is a side effect of how C# configures properties by default and @WildWeazel might have a better solution as the C# expert, but in this case the "boolean getters start with `is`" rule makes it read awkwardly.